src/Sarif/Writers/SarifLogger.cs

```diff
@@ -92,6 +97,19 @@ private static string Redact(string text, IEnumerable<string> tokensToRedact)
             return text;
         }
 
+        private static Uri Redact(Uri uri, IEnumerable<string> tokensToRedact)
+        {
+            if (uri == null) { return uri; }
+
+            string uriText = uri.OriginalString;
+
+            foreach (string tokenToRedact in tokensToRedact)
+            {
+                uriText = uriText.Replace(tokenToRedact, SarifConstants.RemovedMarker);
```

Comment:
>RemovedMarker [](start = 72, length = 13)

Can we call it `RedactedMarked` while we're at it?

Reply:
Good bug. the value of this constant was the old [REMOVED] value which you and Jim updated to [REDACTED] in v2. fixed that as well.

---
In reply to: [214741183](https://github.com/Microsoft/sarif-sdk/pull/1067#discussion_r214741183) [](ancestors = 214741183)